Assets/HoloToolkit/Input/Scripts/GestureManipulator.cs

```diff
@@ -45,69 +46,63 @@ private void Start()
 
         private void OnEnable()
         {
-            if (gestureManager != null)
-            {
-                gestureManager.ManipulationStarted += BeginManipulation;
-                gestureManager.ManipulationCompleted += EndManipulation;
-                gestureManager.ManipulationCanceled += EndManipulation;
-            }
+            gestureManager.OnManipulationStarted += BeginManipulation;
+            gestureManager.OnManipulationCompleted += EndManipulation;
+            gestureManager.OnManipulationCanceled += EndManipulation;
         }
 
         private void OnDisable()
         {
-            if (gestureManager != null)
-            {
-                gestureManager.ManipulationStarted -= BeginManipulation;
-                gestureManager.ManipulationCompleted -= EndManipulation;
-                gestureManager.ManipulationCanceled -= EndManipulation;
-            }
+            gestureManager.OnManipulationStarted -= BeginManipulation;
+            gestureManager.OnManipulationCompleted -= EndManipulation;
+            gestureManager.OnManipulationCanceled -= EndManipulation;
 
             Manipulating = false;
         }
 
-        private void BeginManipulation()
+        private void BeginManipulation(InteractionSourceKind sourceKind)
         {
-            if (gestureManager != null && gestureManager.ManipulationInProgress)
+            // Check if the gesture manager is not null, we're currently focused on this Game Object, and a current manipulation is in progress.
+            if (gestureManager != null && gestureManager.FocusedObject != null && gestureManager.FocusedObject == gameObject && gestureManager.ManipulationInProgress)
```

Comment:
Good point.  We're getting the `Interpolator` component on our game object and making that do our movement, but we still perform the math calculations because we're changing the `Manipulating` flag true.  This prevents calculation when we're not the Focused Object.